flake.nix

```diff
@@ -76,6 +160,10 @@
           {
             config.nixpkgs.overlays = [ self.overlay ];
 
+            # needed to run elastic search
+            config.nixpkgs.config.allowUnfree = true;
```

Comment:
This should not be needed anymore, since we can use the oss versions.

Reply:
I agree, this should be removed. Even if it is required, I think this option should never be set by a module. The decision to include unfree stuff should be made by the user manually.